build.xml

```diff
@@ -527,6 +528,27 @@
         <phingcall target="twig-lint"/>
     </target>
 
+    <target name="upgrade-backend" description="Creates empty file for backend upgrade notes.">
+        <exec executable="php">
+            <arg value="-r"/>
+            <arg value="copy('upgrade-notes/_template.md', 'upgrade-notes/backend_' . date('Ymd_His') . '.md');"/>
+        </exec>
+    </target>
+
+    <target name="upgrade-storefront" description="Creates  empty file named for storefront upgrade notes.">
+        <exec executable="php">
+            <arg value="-r"/>
+            <arg value="copy('upgrade-notes/_template.md', 'upgrade-notes/storefront_' . date('Ymd_His') . '.md');"/>
+        </exec>
+    </target>
+
+    <target name="upgrade-merge" description="Merges all upgrade notes into UPGRADE-{X.X}.md according to structure specified in upgrade-notes/_layout.md. Default value set in version property can be overridden by '-Dversion={X.X}'">
```

Comment:
There should be a note in the releaser to merge the files during release. 

I suggest add phing target call to  `utils/releaser/src/ReleaseWorker/ReleaseCandidate/UpdateUpgradeReleaseWorker.php`

something like `$this->processRunner->run('php phing upgrade-merge');`

Reply:
Added. Thank you for very inspiring CR.